pipeline-adapters/mlrun-pipelines-kfp-v1-8/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[project]
+name = "mlrun-pipelines-kfp-v1-8"
+version = "0.3.10"
+description = "MLRun Pipelines package for providing KFP 1.8 compatibility"
+readme = "README.md"
+requires-python = ">=3.9"
+authors = [
+  { name = "Yaron Haviv", email = "yaronh@iguazio.com" }
+]
+license = { text = "Apache License 2.0" }
+keywords = ["mlrun", "kfp"]
+urls = { homepage = "https://github.com/mlrun/mlrun" }
+dependencies = [
+  "inflection>=0.5.1",
+  "kfp_server_api~=1.8"
+]
+
+[project.optional-dependencies]
+kfp = [
+  "kfp~=1.8.22"
+]
+
+[build-system]
+requires = ["setuptools>=61", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+package-dir = { "" = "src" }
+
+[tool.setuptools.packages.find]
+where = ["src"]
+include = ["mlrun_pipelines"]
+
+[tool.ruff]
```

Comment:
I think that all the ruff/pytest config is unnecessary, same for the other TOML.
These tools will take the config from the root's TOML